commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java

```diff
@@ -0,0 +1,99 @@
+package io.sphere.sdk.orders;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.carts.InventoryMode;
+import io.sphere.sdk.carts.RoundingMode;
+import io.sphere.sdk.carts.TaxPortion;
+import io.sphere.sdk.carts.TaxedPrice;
+import io.sphere.sdk.customergroups.CustomerGroup;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.models.LocalizedEnumValue;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.Price;
+import io.sphere.sdk.products.attributes.AttributeImportDraft;
+import io.sphere.sdk.taxcategories.TaxCategory;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import io.sphere.sdk.types.CustomFieldsDraftBuilder;
+import org.junit.Test;
+
+import javax.money.MonetaryAmount;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static io.sphere.sdk.test.SphereTestUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OrderImportDraftBuilderTest {
+
+    @Test
+    public void createsFromOrderImportDraft() throws Exception {
+        final LocalizedEnumValue yellow = LocalizedEnumValue.of("yellow", LocalizedString.of(ENGLISH, "yellow").plus(GERMAN, "gelb"));
```

Comment:
@katmatt I think that if we use for example `ProductFixtures`, we would have to transform the test into an IntegrationTest, as the already existing methods in the fixture are using the backend to create the products. or what fixtures do you have in mind?